src/runtime/runtime.ts

```diff
@@ -209,10 +234,17 @@ export class Runtime {
 
   // stuff the strategizer needs
 
+  // TBD
+
   // stuff from shells/lib/utils
```

Comment:
Probably just delete this. It's not really very important where it came from.

Reply:
I was following the pattern somebody else established ... otherwise, this would be under 'stuff strategizer needs'. 

What do you suggest?

Reply:
I'd just delete all these comments

Reply:
SGTM